src/ptrace.cc

```diff
@@ -47,6 +48,33 @@ void PtraceDetach(pid_t pid) {
   }
 }
 
+struct user_regs_struct PtraceGetRegs(pid_t pid) {
```

Comment:
I do not have strong feeling about the style here, but FYI the `struct` keyword is only required in C, not C++.